### PR TITLE
fix: Updated Trivy workflow

### PR DESCRIPTION
**Pull request by adkumar1**
- Fixed Trivy workflow. Removed not required steps from workflow
- Updated base image for docker

### CODE REVIEW

File: .github/workflows/trivy.yml
Comment (almadigabor) [inline .github/workflows/trivy.yml:1]: Have you verified these changes with someone from the Security Team?

File: README.md
Comment (almadigabor) [inline README.md:1]: What's the reason of the new chart version? Are you planning a new docker image release?

Comment (adkumar1) [reply README.md:1]: Yes, it consist of new image

File: .github/workflows/trivy.yml
Comment (adkumar1) [reply .github/workflows/trivy.yml:1]: Who can verify them from security

Comment (almadigabor) [reply .github/workflows/trivy.yml:1]: You can always look at this repository. https://github.com/eclipse-tractusx/sig-security

@scherersebastian can you help us out here? Is the changed Trivy workflow fine?

Comment (adkumar1) [reply .github/workflows/trivy.yml:1]: Hi @scherersebastian : Can you please verify trivy workflow

Comment (adkumar1) [reply .github/workflows/trivy.yml:1]: @scherersebastian : Can you please have a look

**Comment by MehranRoshandel** [comment]
@kelaja It seems that Sebastian is not there. Do you know who else can support us here?
We need somebody to commit the PR.
 This PR will solve the Problem in Trivy.

**Review by almadigabor — APPROVED** [review]

**Pull request merged**